pkg/proxy/iptables/proxier.go

```diff
@@ -206,15 +206,36 @@ func NewProxier(ipt utiliptables.Interface, exec utilexec.Interface, syncPeriod
 // It returns true if an error was encountered. Errors are logged.
 func CleanupLeftovers(ipt utiliptables.Interface) (encounteredError bool) {
 	//TODO: actually tear down all rules and chains.
-	args := []string{"-j", "KUBE-SERVICES"}
-	if err := ipt.DeleteRule(utiliptables.TableNAT, utiliptables.ChainOutput, args...); err != nil {
+	args := []string{"-m", "comment", "--comment", "kubernetes service portals"}
+	if err := ipt.DeleteRule(utiliptables.TableNAT, utiliptables.ChainOutput, append(args, "-j", string(iptablesServicesChain))...); err != nil {
```

Comment:
why append() here rather than just put the -j string(...) in the `args :=` statement?